src/structures/GuildChannel.js

```diff
@@ -414,18 +414,21 @@ class GuildChannel extends Channel {
    * @param {string} [options.reason] Reason for cloning this channel
    * @returns {Promise<GuildChannel>}
    */
-  clone({ name = this.name, withPermissions = true, withTopic = true, nsfw, parent, bitrate, userLimit, reason } = {}) {
-    const options = {
-      overwrites: withPermissions ? this.permissionOverwrites : [],
-      nsfw: typeof nsfw === 'undefined' ? this.nsfw : nsfw,
-      parent: parent || this.parent,
-      bitrate: bitrate || this.bitrate,
-      userLimit: userLimit || this.userLimit,
-      reason,
-      type: this.type,
-    };
-    return this.guild.channels.create(name, options)
-      .then(channel => withTopic ? channel.setTopic(this.topic) : channel);
+  clone(options = {}) {
+    if (typeof options.withPermissions === 'undefined') options.withPermissions = true;
+    Util.mergeDefault({
+      name: this.name,
+      overwrites: options.withPermissions ? this.permissionOverwrites : [],
+      withTopic: true,
+      nsfw: this.nsfw,
+      parent: this.parent,
+      bitrate: this.bitrate,
```

Comment:
`this.bitrate` is currently the bitrate in kbits, but `GuildChannelStore#create` expects it to be in bits.

For now simply multiplying with 1000 should do.

Reply:
Bitrate is in bits in the voice rewrite branch. Depending on how long that will take to finish merge. This wont be incorrect for much longer.